tequilapi/endpoints/proposals_test.go

```diff
@@ -89,7 +89,52 @@ func TestProposalsEndpointListByNodeId(t *testing.T) {
         }`,
 		resp.Body.String(),
 	)
-	assert.Equal(t, "0xProviderId", mockProposalProvider.recordedProviderId)
+	assert.Equal(t, "0xProviderId", mockProposalProvider.recordedProviderID)
+}
+
+func TestProposalsEndpointAcceptsAccessPolicyParams(t *testing.T) {
+	mockProposalProvider := &mockProposalProvider{
+		proposals: []market.ServiceProposal{serviceProposals[0]},
+	}
+
+	req, err := http.NewRequest(
+		http.MethodGet,
+		"/irrelevant",
+		nil,
+	)
+	assert.Nil(t, err)
+
+	query := req.URL.Query()
+	query.Set("accessPolicyId", "accessPolicyId")
+	query.Set("accessPolicySource", "accessPolicySource")
+	req.URL.RawQuery = query.Encode()
+
+	resp := httptest.NewRecorder()
+	handlerFunc := NewProposalsEndpoint(mockProposalProvider, &mysteriumMorqaFake{}).List
```

Comment:
`handlerFunc` is very technical. How about:

```go
	proposals := NewProposalsEndpoint(mockProposalProvider, &mysteriumMorqaFake{})
	proposals.List(resp, req, nil)
```

Alternatively, `proposals` could be `resource`, etc. not to confuse with actual proposal representations. But since you're using JSON assert, I guess it doesn't matter here.